### PR TITLE
update_packages method in common.py

### LINKED ISSUE #6: Missing update_packages method
Similar to install_packages, an update_packages method, located in common.py, will be helpful in cases where updates are needed.

### PR DESCRIPTION
**Pull request by dkorn**
Similar to install_packages method, to be used in cases where
updates are needed.

closes #6

### CODE REVIEW

**Comment by dkorn** [comment]
@simon3z in case you haven't noticed

File: projects/common.py [hunk 1]
```diff
@@ -27,11 +27,17 @@ def install_packages(packages, proxy=''):
         sudo('yum -y install {0}'.format(' '.join(packages)), pty=False)
 
 
+def update_packages(packages, proxy=''):
+    with shell_env(http_proxy=proxy, ftp_proxy=proxy):
+        sudo('yum -y update {0}'.format(' '.join(packages)), pty=False)
```
Comment (simon3z) [inline projects/common.py]: Not a big deal, but `install` and `update` really look the same so they should be unified, maybe something like:

```
def install_packages(packages):
    yum_command('install', packages)

def update_packages(packages):
    yum_command('update', packages)

def yum_command(command, packages):
    ...
```

The reason for having `install_packages` and `update_packages` is that they encapsulate the commands and in the future we'll be able to switch method used to install/update (yum, dnf, whatever...).

Comment (simon3z) [inline projects/common.py]: Ah we should also move the "proxy" argument from being an argument to being a fabric `env` variable.

Comment (dkorn) [inline projects/common.py]: I added the yum command as you offered, thanks!

regarding the proxy support - you said you'll move it to the env.

**Pull request merged**